test/solvers.jl

```diff
@@ -60,6 +60,11 @@ mos && push!(ip_solvers, Mosek.MosekSolver(LOG=0))
 cbc && push!(ip_solvers, Cbc.CbcSolver(logLevel=0))
 glp && push!(ip_solvers, GLPKMathProgInterface.GLPKSolverMIP())
 #osl && push!(ip_solvers, CoinOptServices.OsilSolver())
+# IP solvers that give duals for relaxations
```

Comment:
Meh, these hacks will be fixed by https://github.com/JuliaOpt/MathProgBase.jl/issues/156